pkg/utils/utils.go

```diff
@@ -168,15 +169,11 @@ func RemoveDuplicates(elements []string) []string {
 
 // GetConfigFilePath returns the path to the Scaleway CLI config file
 func GetConfigFilePath() (string, error) {
-	homeDir := os.Getenv("HOME") // *nix
-	if homeDir == "" {           // Windows
-		homeDir = os.Getenv("USERPROFILE")
-	}
-	if homeDir == "" {
-		return "", errors.New("user home directory not found")
+	usr, err := user.Current()
```

Comment:
`user.Current()` forbid the creation of static binaries when cross compiling, so we need to use the oldschool way :)